bval-jsr/pom.xml

```diff
@@ -209,9 +209,11 @@
         <dependency>
             <groupId>org.apache.tomcat</groupId>
             <artifactId>tomcat-jasper-el</artifactId>
-            <version>10.1.24</version>
+            <version>11.0.0-M21</version>
```

Comment:
even if for tests we should stick to a prod version with no unexpected feature -> 10.1.25 (everywhere indeed)

Reply:
agree but same argument can also be applied to TCK tests themselves (note the staging repo, they're not properly released yet)

Reply:
downgraded to tomcat 10.1 for now, expressly is also only EL 5 right now anyways

Reply:
any idea why we have both, totally missed that earlier but does not make sense to me to have expressly there? :thinking:

Reply:
guess because of https://github.com/jungm/bval/blob/e5559c0571809f2445e3ec730e7ecbbcca480a87/bval-jsr/src/test/java/org/apache/bval/jsr/DefaultMessageInterpolatorTest.java, it tests against both jasper and expressly